reference/settings.v3.yml

```diff
@@ -1359,6 +1360,63 @@ paths:
                     type: object
       tags:
         - Storefront Status
+  '/stores/{store_hash}/v3/settings/inventory':
+    get:
+      summary: Get Inventory Settings
+      tags:
+        - Inventory
+      responses:
+        '200':
+          description: 'OK, null indicates that a particular field has not been overridden on a channel level when channel level settings are requested'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/InventorySettings'
+                  meta:
+                    type: object
+        '422':
+          description: Provided settings could not be applied for some reason - detailed errors in the response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+      parameters:
+        - $ref: '#/components/parameters/ChannelIdParam'
+      description: Get Inventory Settings
+    put:
+      summary: Update Inventory Settings
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InventorySettings'
+        description: null set for a particular field removes override on a channel level and means inheritance from a global level
```

Comment:
```suggestion
        description: null set for a particular field removes override on a channel level and means inheritance from a global level
```
I didn't see channel level inventory notification settings in my UI

Reply:
It might not have been available in the UI way back in Sept, but it is there now at `https://store-{HASH}.mybigcommerce.com/manage/channel/{channel_id}/notifications-settings`

![image](https://user-images.githubusercontent.com/29524703/207418641-c8ef13e6-f095-423e-af3d-28a641f61176.png)